frontend/src/scenes/max/maxLogic.tsx

```diff
@@ -73,99 +47,74 @@ export const maxLogic = kea<maxLogicType>([
     })),
 
     actions({
-        askMax: (prompt: string, generationAttempt: number = 0) => ({ prompt, generationAttempt }),
-        stopGeneration: true,
-        completeThreadGeneration: (testOnlyOverride = false) => ({ testOnlyOverride }),
-        setThreadLoading: (isLoading: boolean) => ({ isLoading }),
-        addMessage: (message: ThreadMessage) => ({ message }),
-        replaceMessage: (index: number, message: ThreadMessage) => ({ index, message }),
-        setThread: (thread: ThreadMessage[]) => ({ thread }),
-        setMessageStatus: (index: number, status: MessageStatus) => ({ index, status }),
         setQuestion: (question: string) => ({ question }),
         setVisibleSuggestions: (suggestions: string[]) => ({ suggestions }),
         shuffleVisibleSuggestions: true,
-        retryLastMessage: true,
         scrollThreadToBottom: (behavior?: 'instant' | 'smooth') => ({ behavior }),
         setConversationId: (conversationId: string) => ({ conversationId }),
-        setConversation: (conversation: Conversation) => ({ conversation }),
-        setTraceId: (traceId: string) => ({ traceId }),
-        resetThread: true,
-        cleanThread: true,
         startNewConversation: true,
         toggleConversationHistory: (visible?: boolean) => ({ visible }),
         loadThread: (conversation: ConversationDetail) => ({ conversation }),
```

Comment:
It's now clear what place `maxGlobalLogic` and `maxThreadLogic` have, but not clear what `maxLogic` is. Should we now merge the remains of `maxLogic` into `maxGlobalLogic`?